src/confluent_kafka/src/Consumer.c

```diff
@@ -1082,20 +1173,23 @@ static PyMethodDef Consumer_methods[] = {
 	  METH_VARARGS|METH_KEYWORDS,
 	  ".. py:function:: consume([num_messages=1], [timeout=-1])\n"
 	  "\n"
-	  "  Consume messages, calls callbacks and returns list of messages "
-	  "(possibly empty on timeout).\n"
+	  "  Consume a list of messages (possibly empty on timeout). Callbacks may "
```

Comment:
"Consume a list of messages" sounds like it is consuming from a list, should probably say that it returns a list of messages.